__tests__/integration/mirador/blank.html

```diff
@@ -13,7 +13,9 @@
     <script type="text/javascript">
      var miradorInstance = Mirador.viewer({
        id: 'mirador',
-       windows: [],
+       windows: [{
+         manifestId: 'https://www.e-codices.unifr.ch/metadata/iiif/gau-Fragment/manifest.json'
```

Comment:
Can we remove the added manifest here, if needed we can create a new page "multiplesequence.html" or something similar?

Reply:
Hey, just pushed